src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 9 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/ExperimentalFeature/Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```
ForEach-Object -Parallel common parameters.Verifies that ErrorAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:128 char:40
    + ... k = { 1..1 | ForEach-Object -Parallel { "Hello" } -ErrorAction Stop }
    +                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that WarningAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:132 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -WarningAction Silen ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that InformationAction common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:136 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -InformationAction S ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```
ForEach-Object -Parallel common parameters.Verifies that PipelineVariable common parameter is not supported
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand' to be thrown, but the FullyQualifiedErrorId was 'AmbiguousParameterSet,Microsoft.PowerShell.Commands.ForEachObjectCommand'. from /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:140 char:40
    + ...  = { 1..1 | ForEach-Object -Parallel { "Hello" } -PipelineVariable pi ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 182
182:         { & $scriptBlock } | Should -Throw -ErrorId 'ParallelCommonParametersNotSupported,Microsoft.PowerShell.Commands.ForEachObjectCommand'
```

Reply:
@KirkMunro  Experimental features are now turned on for preview builds by default, so the change you made to the metadata.json file is no longer needed.  I don't know why these errors are occurring, but can you try removing your metadata.json file changes and see if that resolves the problem?

Reply:
There are a few issues here:
1. Experimental features do not appear to be properly turned on by default, because if they were, these errors wouldn't happen.
1. The `ForEach-Object -Parallel` Pester tests fail because the `PSForEachObjectParallel` experimental feature is disabled and part of the Pester tests added in #10229 failed to include the required logic to skip those tests if the experimental feature is disabled. I just corrected this in PR #10354. Once the tests in that PR pass, if we can get that merged, that will make the majority of the Pester failures here go away.
1. The Pester failure that indicates that experimental features are not enabled will continue to fail because experimental features are not enabled properly in the CI builds.

Reply:
If reverting your metadata.json file changes does not solve the problem, then I feel we should remove the experimental feature restriction entirely from this PR.  I would like to see people begin experimenting with this in the next preview if possible.

Reply:
I don't mind doing that at all. In fact, I think the way experimental features are being pushed right now is a little misguided, and that they should only be used for breaking changes, or for net new functionality that is either still under active development (i.e. not ready for production yet) or that has public facing interfaces (cmdlets, new parameter sets, new parameters) whose design is still in question; however, I've heard from @SteveL-MSFT and the PowerShell Committee that: "By default, our expectation is that new features are put behind an Experimental Flag...".

Given the pushback, and to save myself from going back and forth with code changes, I would like confirmation that the experimental nature of this PR can be removed before I go ahead and remove it.

Reply:
I'm taking a look

Reply:
https://github.com/PowerShell/PowerShell/pull/10354 is expected to fix this or not?

Reply:
#10354 is expected to fix the bulk of the Pester test issues. There are other Pester test issues that will remain once #10354 is merged in.

It is clear from the Pester test results that Experimental Features are _not_ enabled by default in the CI environment. For example, the first failed test on each platform that indicates that "PSCommandNotFoundSuggestion" is not enabled. That fails but as far as I can tell it has nothing to do with this PR.